server/apps/immich/src/api-v1/asset/asset-repository.ts

```diff
@@ -285,25 +285,22 @@ export class AssetRepository implements IAssetRepository {
       },
       select: ['deviceAssetId'],
     });
-    const res: string[] = [];
+    const res: (string | null)[] = [];
```

Comment:
Probably better to filter out null values if there's no reason to keep them included